src/bundle/Resources/config/services/suggestions.yaml

```diff
@@ -0,0 +1,32 @@
+parameters:
+    ibexa.site_access.config.default.search.suggestion.min_query_length: 3
+    ibexa.site_access.config.default.search.suggestion.result_limit: 5
+
+services:
+    _defaults:
+        autoconfigure: true
+        autowire: true
+        public: false
+
+    Ibexa\Bundle\Search\Controller\SuggestionController:
```

Comment:
```suggestion
    Ibexa\Bundle\Search\Controller\SuggestionController:
        tags:
            - { name: container.service_subscriber, key: serializer, id: ibexa.search.serializer.suggestion_serializer }
```

Reply:
I have configured it like this. That solution doesn't work as expected - the controller still uses the default serializer. 

Neither works 

```
public static function getSubscribedServices(): array
{
    return array_merge(parent::getSubscribedServices(), [
        'serializer' => 'ibexa.search.serializer.suggestion_serializer',
    ]);
}
```

```
"Ibexa\Bundle\Search\Controller\SuggestionController::getSubscribedServices()" must return valid PHP types for service "Ibexa\Bundle\Search\Controller\SuggestionController" key "serializer", "ibexa.search.serializer.suggestion_serializer" returned.
```